### DISCUSSION TITLE
Creating Python Package

### DISCUSSION THREAD

**Discussion by kingabzpro** [martinm07/tokenization-layer#3] [Ideas]
I have this idea to convert your code into python package even though it's not working yet but for future reference.

**Comment by martinm07** [comment]
Yes, perhaps. I'll look into it and see what I can do.

**Comment by martinm07** [comment]
Well, after a lot longer than I care to admit, the package is finished :)
[Here](https://pypi.org/project/tokenization-layer/0.0.1/) is the page on PyPi.
I also created documentation for the package [here](https://martin-github07.gitbook.io/tokenization-layer/), along with the synchronized GitHub repository [here](https://github.com/martinm07/tokenization-layer-docs) (so that anyone can contribute to the docs :) ).
I hope it works okay...

**Reply by kingabzpro** [reply]
I men Just waoooooooooooooooo, you have exceeded my expectations. I would love you to post your progress on community. Also share your work on twitter and LinkedIn by using hashtags.

**Reply by kingabzpro** [reply]
Don't forget to Post on reedit machine learning and data science. Also If you have time try making account on https://www.polywork.com/ you can use my VIP code : kingabzpro-jigglypuff.

You have done all the hard work. Now its time to sell your hard work to the people. Just like when you publish book, you go on tours, have conventions, post on social media .

**Reply by martinm07** [reply]
Haah... Thank you for your kind words! I'll look into it :)
